src/cli.rs

```diff
@@ -465,6 +465,27 @@ fn subcommand_scaffold() -> Command {
             .help("Policy URI or SHA prefix. Supported schemes: registry://, https://, file://. If schema is omitted, file:// is assumed, rooted on the current directory."),
     );
 
+    let mut vap_args = vec![
+        Arg::new("cel-policy")
+            .long("cel-policy")
+            .value_name("URI")
+            .default_value("ghcr.io/kubewarden/policies/cel-policy:latest")
```

Comment:
I would be ok with "ghcr.io/kubewarden/policies/cel-policy:<version>" so by default the user is forced to edit and doesn't instantiate a possible broken policy.